### ISSUE TITLE
Transfer of Library is in reversed order 

### ISSUE THREAD

**Issue by nikoelt** [linsomniac/spotify_to_ytmusic#102]
Hey Folks

Not sure if this is a user error or not
I used your amazing scripts and the transfer from my Spotify Account to my YouTube Music account went flawlessly

I couldn't help but to notice that my Library/Liked Songs were in completely reversed order and unlike a playlist there is no way to reverse/change order of Liked Songs in YouTube Music. 

The way that the app scrolls on iOS is also quite annoying as for me it lacks fast scroll and you're stuck dong the 'making it rain' gesture to get to the latest songs I used to enjoy on Spotify

After the install procedure I simply used `s2yt_load_liked` inside MacOS Terminal on my M1 MBP

Is there something I did wrong? What can I do to rectify this?

**Comment by sctech-tr** [comment]
i don't think this is a user error. it happened to me too while transferring. oh well

**Comment by killerbytes** [comment]
you need to add `reverse_playlist=False` argument

**Comment by fredso104** [comment]
> you need to add `reverse_playlist=False` argument

Hi, noob here, how do I go about doing this?

**Comment by patriciasium** [comment]
hi, where do we specifically insert the reverse_playlist=False line?

**Comment by DrekAnon** [comment]
> you need to add `reverse_playlist=False` argument

where?

**Comment by ypurpl** [comment]
https://github.com/linsomniac/spotify_to_ytmusic/issues/175#issuecomment-2784797914
